enclave_manager/Dockerfile

```diff
@@ -171,19 +171,21 @@ RUN mkdir -p /opt/intel \
  && mkdir -p /etc/init
 WORKDIR /opt/intel
 
-# Install Intel SGX common library
-RUN wget https://download.01.org/intel-sgx/sgx-linux/2.7.1/distro/ubuntu18.04-server/libsgx-enclave-common_2.7.101.3-bionic1_amd64.deb \
- && dpkg -i libsgx-enclave-common_2.7.101.3-bionic1_amd64.deb
+RUN wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-enclave-common/libsgx-enclave-common_2.10.100.2-bionic1_amd64.deb \
+ && wget https://download.01.org/intel-sgx/latest/linux-latest/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-epid/libsgx-epid_2.10.100.2-bionic1_amd64.deb \
+ && wget https://download.01.org/intel-sgx/latest/linux-latest/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-launch/libsgx-launch_2.10.100.2-bionic1_amd64.deb \
+ && wget https://download.01.org/intel-sgx/latest/linux-latest/distro/ubuntu18.04-server/debian_pkgs/libs/libsgx-urts/libsgx-urts_2.10.100.2-bionic1_amd64.deb \
+ && dpkg -i *
 
-# Install Intel SGX SDK
-RUN SGX_SDK_FILE=sgx_linux_x64_sdk_2.7.101.3.bin \
- && wget https://download.01.org/intel-sgx/sgx-linux/2.7.1/distro/ubuntu18.04-server/$SGX_SDK_FILE \
- && echo "yes" | bash ./$SGX_SDK_FILE \
- && rm $SGX_SDK_FILE \
- && echo ". /opt/intel/sgxsdk/environment" >> /etc/environment
+RUN SGX_SDK_FILE=sgx_linux_x64_sdk_2.10.100.2.bin \
+  && wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/ubuntu18.04-server/$SGX_SDK_FILE \
+  && echo "yes" | bash ./$SGX_SDK_FILE \
+  && rm $SGX_SDK_FILE \
+  && rm -f /usr/lib/x86_64-linux-gnu/libsgx_urts.so \
```

Comment:
Do we need to remove `/usr/lib/x86_64-linux-gnu/libsgx_urts.so` ? Could you please add a comment why we need to remove this library

Reply:
No need to remove. I will delete the line.